ultralytics_yolov5.md

```diff
@@ -4,74 +4,68 @@ background-class: hub-background
 body-class: hub
 category: researchers
 title: YOLOv5
-summary: YOLOv5 in PyTorch > ONNX > CoreML > TFLite
+summary: Ultralytics YOLOv5 🚀for object detection, instance segmentation and image classification.
 image: ultralytics_yolov5_img0.jpg
 author: Ultralytics
 tags: [vision, scriptable]
 github-link: https://github.com/ultralytics/yolov5
 github-id: ultralytics/yolov5
-featured_image_1: ultralytics_yolov5_img1.jpg
+featured_image_1: ultralytics_yolov5_img1.png
 featured_image_2: ultralytics_yolov5_img2.png
 accelerator: cuda-optional
-demo-model-link: https://huggingface.co/spaces/pytorch/YOLOv5
+demo-model-link: https://hub.ultralytics.com/projects/DnIqQCdlI37oliWBAoQJ
```

Comment:
Note: this changes the URL from HF spaces to the ultrlytics namespace. Functionality looks about the same tho, so that's fine as far as I'm concerned. CC @vmoens